### ISSUE TITLE
observableMiddleware usage?

### ISSUE THREAD

**Issue by matthewwithanm** [acdlite/redux-rx#10]
The observableMiddleware accepts a `next` function, but redux's [`applyMiddleware`](https://github.com/rackt/redux/blob/611a543603b824345e84ed1d64c08f783d601a02/docs/api/applyMiddleware.md) expects a function like that accepts `{dispatch: Function, getState: Function}` and returns a function like redux-rx's `observableMiddleware`.

Are users expected to wrap observableMiddleware themselves (e.g. `const realMiddleware = () => observableMiddleware;`)?

Any documentation about its usage would be appreciated.

Thanks!

**Comment by acdlite** [comment]
Good catch. You're right — the observable middleware predates the new, updated middleware signature required by `applyMiddleware()`. I'd accept a PR to fix this, or I'll get to it soon :)
